CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Release transport resources of nodemailer after send email

```

Comment:
Is there any issue number associated? (I guess no, but to confirm...)

Reply:
No issue was opened